src/sentry/tasks/commits.py

```diff
@@ -93,10 +93,16 @@ def fetch_commits(release_id, user_id, refs, prev_release_id=None, **kwargs):
             )
             continue
 
-        try:
-            provider_cls = bindings.get('repository.provider').get(repo.provider)
-        except KeyError:
-            continue
+        if repo.provider.startswith('integrations:'):
```

Comment:
i think to be safe you should do `repo.provider and repo.provider.startswith` to make sure it isn't `None`